apps/www/_alternatives/supabase-vs-heroku-postgres.mdx

```diff
@@ -0,0 +1,77 @@
+
+- Auto-generated API - [never write an API again](https://supabase.com/docs/guides/api#rest-api-2)
+- Realtime - [subscribe to data changes via websockets](https://supabase.com/docs/reference/dart/subscribe)
+- Auth - [users can log in and out of your application](https://supabase.com/auth)
+- Functions - [deploy custom logic to the edge](https://supabase.com/edge-functions)
+- Storage - [serve large files and folders](https://supabase.com/storage)
+- PgBouncer - [connection pooling useful for serverless computing](https://supabase.com/docs/guides/database/connecting-to-postgres#connection-pool)
+
+## How are they similar?
+
+Heroku Postgres and Supabase both offer:
+
+- [A web UI for managing your instance](https://supabase.com/docs/guides/database#table-view)
+- SLAs and Enterprise-grade support packages
+- Direct SSL connections to Postgres
+- Postgres Extensions (see [Supabase Extensions](https://supabase.com/docs/guides/database/extensions), [Heroku Extensions](https://devcenter.heroku.com/articles/heroku-postgres-extensions-postgis-full-text-search))
+- [Backups and PITR](https://supabase.com/blog/2020/08/02/continuous-postgresql-backup-walg) (not on Heroku’s Hobby tier)
+- [Postgres logs](https://supabase.com/docs/guides/platform/logs) (not on Heroku’s Hobby tier)
+- Encryption-at-rest (not on Heroku’s Hobby tier)
+
+## What are the differences?
+
+### Core Features
+
+Both solutions run PostgreSQL, but in a time when Developer Experience matters, there is a lot you can do to improve the speed at which developers can build products faster and with less human resource. These are some of the key differences between Heroku Postgres and Supabase in terms of features:
+
+- Supabase is more than just the raw database, it also comes with:
+  - [Connection pooling](https://supabase.com/docs/guides/database/connecting-to-postgres#connection-pool) so that you won’t run out of connections in a serverless environment.
+  - [Auto-generated APIs](https://supabase.com/docs/guides/api#rest-api-2) based on your schema, so you can communicate with your database directly from the client.
+  - [Realtime API](https://supabase.com/docs/reference/dart/subscribe) is useful for when you want to subscribe to changes to your database over websockets.
+  - [Auth API](https://supabase.com/auth) can be used to leverage Postgres’s Row Level Security model, and control access to sensitive data on a per user, or per group level.
+  - [Functions](https://supabase.com/edge-functions) can be deployed out to the edge directly from the Supabase CLI, which means you can run sensitive business logic or transformations in a serverless fashion.
+  - [File Storage](https://supabase.com/storage) is useful for when your app needs to store large files and folders that aren’t suitable for storing within Postgres itself.
+  - A spreadsheet-like web interface for building your schemas and inspecting data.
```

Comment:
Above it says that Heroku has a webUI - do we want to call out what their web UI does?